vspace/src/main/webapp/WEB-INF/views/staff/modules/sequences/sequence.jsp

```diff
@@ -1,23 +1,200 @@
-<%@ page pageEncoding="UTF-8" %>
+<%@ page pageEncoding="UTF-8"%>
 <%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
-<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/functions" prefix="fn"%>
 <%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
 
 <ol class="breadcrumb">
-  <li class="breadcrumb-item"><a href="<c:url value="/staff/dashboard" />">Dashboard</a></li>
-  <li class="breadcrumb-item"><a href="<c:url value="/staff/module/list" />">Modules</a></li>
-  <li class="breadcrumb-item"><a href="<c:url value="/staff/module/${module.id}" />">${module.name}</a></li>
-  <li class="breadcrumb-item active">${sequence.name}</li>
+	<li class="breadcrumb-item"><a
+		href="<c:url value="/staff/dashboard" />">Dashboard</a></li>
+	<li class="breadcrumb-item"><a
+		href="<c:url value="/staff/module/list" />">Modules</a></li>
+	<li class="breadcrumb-item"><a
+		href="<c:url value="/staff/module/${module.id}" />">${module.name}</a></li>
+	<li class="breadcrumb-item active">${sequence.name}</li>
 </ol>
 
-<h1>Sequence: ${sequence.name}</h1>
+
+
+<script>
+
+
+$(document).ready(function() { 
+ 
+	
+    //-------- edit contentblock description Sequence starts --------
+    $("#submitDescription").hide()
+    $("#cancelEditDescription").hide()	
+    
+    $("#editDescription").click(function() {
+    	var description = $("#description").text()
+    	document.getElementById("description").value = description;
+        $('<textarea id="newDescription" style="margin-top: 1%;" class="form-control" type="text">'+description+'</textarea>').insertBefore( "#description" );
+        $("#description").hide()
+        $("#newDescription").val(description)
+        $("#editDescription").hide()
+        $("#submitDescription").show()
+        $("#cancelEditDescription").show()  
+        
+    });
+
+    
+    // --- Submit Description ------
+    
+    $("#submitDescription").click(function() {
+        var formData = new FormData();
+        formData.append('description', $("#newDescription").val());
+        $.ajax({
+        url: "<c:url value="/staff/module/${module.id}/sequence/${sequence.id}/edit/description?${_csrf.parameterName}=${_csrf.token}" />",
+        type: 'POST',
+        cache       : false,
+        contentType : false,
+        processData : false,
+        data: formData,
+        enctype: 'multipart/form-data',
+        success: function(data) {
+            // replace text box with new description
+            $("#submitDescription").hide()
+            $("#cancelEditDescription").hide()
+            $("#editDescription").show()
+            var val = $("#newDescription").val();
+            $('<p id="description"style="margin-top: .5rem; margin-bottom: .5rem;">val</p>').insertBefore( "#newDescription" );
```

Comment:
why are you inserting a new element with id "description"? the old one is only hidden, isn't it?

Reply:
I have referred to the functionality implemented for Slides.jsp for it's title and description and have incorporated it for sequence.

Reply:
comment still applied. if you end up with multiple elements with id "description" something is wrong.

Reply:
okay, I will use different id names, I tried with the one that is hidden, but it's not working.

Reply:
it's not about using a different id but about reusing the existing element.